Document/0x05b-Basic-Security_Testing.md

```diff
@@ -41,29 +41,50 @@ Note: On Linux, you need to choose an SDK directory. `/opt`, `/srv`, and `/usr/l
+| Reset                      | Can be restored to factory settings or reflashed.                                                                                                                                                                                                            | Emulators can be deleted and recreated.                                                                                                               |
+| Snapshots                  | Not possible.                                                                                                                                                                                                                                                | Supported, great for malware analysis.                                                                                                                |
+| Speed                      | Much faster than emulators.                                                                                                                                                                                                                                  | Typically slow, but improvements are being made.                                                                                                      |
+| Cost                       | Typically start at $200 for a usable device. You may require different devices, such as one with or without a biometric sensor.                                                                                                                              | Only storage costs.                                                                                                                                   |
+| Ease of rooting            | Highly dependent on the device.                                                                                                                                                                                                                              | Typically rooted by default.                                                                                                                          |
+| Ease of emulator detection | It's not an emulator, so very easy.                                                                                                                                                                                                                          | Many artefacts will exist, making it easy to detect that the app is running in an emulator.                                                           |
```

Comment:
It's not an emulator, so very easy. -> It's not an emulator, so emulator checks are not applicable.

Reply:
Fixed